test/game.js

```diff
@@ -0,0 +1,93 @@
+const { expect } = require("chai");
+
+describe("Game", function () {
+  let Token;
+  let token;
+  let Game;
+  let game;
+  let owner;
+  let player;
+  let judge;
+  let addrs;
+
+  beforeEach(async function () {
+    [owner, player, judge, ...addrs] = await ethers.getSigners();
+
+    Token = await ethers.getContractFactory("Fweb3");
+    token = await Token.deploy();
+
+    Game = await ethers.getContractFactory("Game");
+    game = await Game.deploy(token.address);
+
+    // Seed everyone with enough tokens
+    await token.transfer(game.address, ethers.utils.parseEther("1000000"));
+    await token.transfer(owner.address, ethers.utils.parseEther("15"));
+    await token.transfer(player.address, ethers.utils.parseEther("15"));
+    await token.transfer(judge.address, ethers.utils.parseEther("15"));
+
```

Comment:
Add a line to give the game.address an allowance so it have 10000 tokens to play with or something.